feat(projectcard component): i have incorporated a more isomorphic de…

arunpariyar opened the pull request:
…sign for the project cards

BREAKING CHANGE: It is specific to the mock data and project card component

arunpariyar committed:
feat(projectcard component): i have incorporated a more isomorphic design for the project cards

BREAKING CHANGE: It is specific to the mock data and project card component

n4mr3g commented on src/types/Project.tsx:
```diff
@@ -1,6 +1,8 @@
+  title: string;
+  create_date: Date;
+  intro: string;
+  tech: string[];
```
maybe tech**s** (in plural) could be a better name, as it is an array? or something else more descriptive, like 'techStacks'

arunpariyar replied:
renamed to techStacks :)

n4mr3g commented on src/types/Project.tsx:
```diff
@@ -1,6 +1,8 @@
+  id: number;
+  author: string;
+  title: string;
+  create_date: Date;
```
maybe 'creationDate' or 'createdAt' would be better name options

n4mr3g commented on src/styles/ProjectCard.module.css:
```diff
@@ -1,8 +1,38 @@
+  font-size: 12px;
+}
+
+.techAndDate {
```
What about changing this selector to something like 'project-info', 'project-details'?

arunpariyar replied:
renamed it to simply details

n4mr3g commented on src/styles/ProjectCard.module.css:
We should probably keep CSS selectors in kebab-case

n4mr3g replied:
Resolved: we allow camelCase due to how CSS modules work in Next.js

arunpariyar committed:
refactor(projects page and project card): add style to the new project button

arunpariyar committed:
refactor(project button): add unique content and style to the new project button

changes made here is specific to the project page and button component is now improved

n4mr3g reviewed (approved):
LGTM

n4mr3g committed:
Merge branch 'dev' into style/ap/issue-18/styling-the-project-cards

n4mr3g commented:
Solve merge conflicts
